xcodeproj/internal/default_input_file_attributes_aspect.bzl

```diff
@@ -100,11 +99,16 @@ def _default_input_file_attributes_aspect_impl(target, ctx):
         infoplists = ("infoplists")
     elif AppleFrameworkImportInfo in target:
         xcode_targets = {"deps": [target_type.compile]}
+        should_generate_target = False
```

Comment:
Doesn't this get overridden on line 107?

```
should_generate_target = executable and not executable.is_source
```

Reply:
107 is in an `else` statement, so no.

Reply:
Ah yes, gotta love semantic white space.